### PR TITLE
content/WIP add new HC metas

### PR DESCRIPTION
**Pull request by marybyrnepark**
Add metas to top 20 HC pages

### CODE REVIEW

**Comment by bethpotts** [comment]
@1Copenut Thanks for the review and merge!

File: content/pages/health-care/apply.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ plainlanguage: 11-02-16 certified in compliance with the Plain Writing Act
 lastupdate: 2017-08-28
 collection: healthCare
 order: 3
-description: Applying for VA health care benefits is easy. Find out which documents you’ll need, and start your online application today.
+description: Find out what documents you'll need to apply for VA health care benefits. Apply online for VA health care benefits today.
```
Comment (1Copenut) [inline content/pages/health-care/apply.md]: @bethpotts Removed a second `description` block here, and confirmed the correct one was added. Verified the page builds correctly on localhost, and all build tests are now passing.

**Review by 1Copenut — COMMENTED** [review]
LGTM. I tested all updated pages locally for proper `meta:description` tags and proper layout and rendering.

**Review by 1Copenut — APPROVED** [review]
LGTM!

**Pull request merged**